src/plot_api/plot_api.js

```diff
@@ -2762,11 +2762,6 @@ function makePlotFramework(gd) {
     fullLayout._glcontainer.enter().append('div')
         .classed('gl-container', true);
 
-    fullLayout._geocontainer = fullLayout._paperdiv.selectAll('.geo-container')
-        .data([0]);
-    fullLayout._geocontainer.enter().append('div')
```

Comment:
An artefact from the first iteration back in July 2015 - where I found that [`Fx.lonehover`](https://github.com/plotly/plotly.js/blob/143749ecfecf676adfaa42ff03469e273c3f8cea/src/plots/cartesian/graph_interact.js#L752-L808) worked best off a plain `<div>` (fast-forward today, this not longer the case).